test/controllers/games_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class GamesControllerTest < ActionController::TestCase
```

Comment:
While I admittedly prefer `Test::Unit`, FHP has settled on rspec. As such, this should be in the specs folder using Rspec syntax.

That said - we can leave final decision up to the team (@THP-nov-2016) as to which we use, but it can only be one.